appium/webdriver/extensions/location.py

```diff
@@ -29,24 +29,25 @@ def toggle_location_services(self):
         self.execute(Command.TOGGLE_LOCATION_SERVICES, {})
         return self
 
-    def set_location(self, latitude, longitude, altitude):
+    def set_location(self, latitude, longitude, altitude=None):
         """Set the location of the device
 
         Args:
-            latitude (float): String or numeric value between -90.0 and 90.00
-            longitude (float): String or numeric value between -180.0 and 180.0
-            altitude (float): String or numeric value
+            latitude (Union[float, str]): String or numeric value between -90.0 and 90.00
```

Comment:
`Union` doesn't work for type hinting on pycharm, but it's correct.

https://mypy.readthedocs.io/en/latest/kinds_of_types.html#union-types
https://www.python.org/dev/peps/pep-0484/#union-types